backend/LeetWars.Core/Dockerfile

```diff
@@ -1,5 +1,13 @@
 FROM mcr.microsoft.com/dotnet/aspnet:6.0-focal AS base
 ARG PROJECT_PORT
+ARG BLOB_URL
+ARG BLOB_CONTAINER_NAME
+ARG BLOB_ACCESS
+
+ENV BLOB_URL=$BLOB_URL
+ENV BLOB_CONTAINER_NAME=$BLOB_CONTAINER_NAME
+ENV BLOB_ACCESS=$BLOB_ACCESS
+
```

Comment:
do we really need this? we can store all secret data in env variables, not secret in appsettings